Source/Evolve/WarpXEvolve.cpp

```diff
@@ -80,6 +80,7 @@ WarpX::Evolve (int numsteps)
     const int step_begin = istep[0];
     for (int step = istep[0]; step < numsteps_max && cur_time < stop_time; ++step)
     {
+        WARPX_PROFILE_REGION_START("WarpX::Evolve::step");
```

Comment:
Wait is `WARPX_PROFILE` not sufficient here?

Reply:
I specifically added and used PROFILE_REGION_START and REGION_STOP because START and STOP do not add a region to TinyProfiler outputs, only Full Profiler outputs. So, this change would not impact the output of most users runs. (Too many regions can make parsing the output impossible.)

They can be easily changed to a PROFILE_REGION (see line 64 for the only REGION example so far) if you also want an isolated TInyProfiler region out for each sub-section you instrument.